src/command_modules/azure-cli-iotcentral/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.3
++++++
+* Added template and display name options for IoT Central Application creation.
+
```

Comment:
Can you add the breaking change to the history?
Ex: https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-acr/HISTORY.rst#210

Reply:
Done

Reply:
@williexu It's saying the azure-cli and azure-cli-core versions are invalid. Do i need to add the breaking change there too?

Reply:
can you rebase off of the dev branch?

Reply:
done

Reply:
hm why does this show all of these other commits that are already in dev?

Reply:
did you rebase or did you merge?....
it looks like your forked branch has diverged from dev significantly

Reply:
i rebased. im not sure what happened...

Reply:
@williexu good to merge?